scissors/src/main/java/com/lyft/android/scissors/CropView.java

```diff
@@ -301,6 +304,28 @@ public LoadRequest using(@Nullable BitmapLoader bitmapLoader) {
             return new LoadRequest(cropView).using(bitmapLoader);
         }
 
+        @IntDef({LOADER_INVALID, LOADER_PICASSO, LOADER_GLIDE, LOADER_UIL})
```

Comment:
From API standpoint, Enums are more discoverable and safe for the users of the library, we will merge it as is but will change it to enums in a separate PR :)